### PR TITLE
Additional illustrative gallery examples for new fitting module

### PR DESCRIPTION
**Pull request by TranHuuNhatHuy**
This pull request aims to implement all illustrative scripts serving as gallery examples for the new fitting module added in #520. This pull request will feature:

- `plot_newfitting_Tgas-molfrac.py` : Mr. @minouHub 's user-testing case of CO absorbance spectrum near 2011 cm-1. This case features spectrum extraction from a `.mat` MATLAB file. Originally, this file was 1.2 MB, quite large to be added to RADIS. Thus, I removed all fields unnecessary for spectrum generation, and now it only has around 400 kB left. The MATLAB file is `trimmed_1857_VoigtCO_Minesi.mat`.
- `plot_newfitting_comparison_oldnew.py` : performance comparative example between [current 1-temperature fitting](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py) and my new fitting module, under exactly the same ground-truths and settings.
- `plot_newfitting_comparison_methods.py` : a benchmarking example which compares performance between different LMFIT fitting algorithms.

Also, this pull request addresses several performance evaluations of the new fitting module, their results and gallery examples:

1. Comparison with the old [1-temperature fitting example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py)
Conducted with `plot_newfitting_comparison_oldnew.py`. The benchmark result shows that, under exactly the same conditions, the new best fitted value differ 0.45% from the old one (`1464.1 K` from the old `1457.5 K`). New fitting module requires half as many iterations as the old one and hence faster, with much smaller residual. More details can be seen [here](https://github.com/radis/radis/pull/522#issuecomment-1240328468).

2. Comparison among different fitting methods
Conducted with `plot_newfitting_comparison_methods.py`, among 17 [LMFIT methods](https://lmfit.github.io/lmfit-py/fitting.html#choosing-different-fitting-methods), targeting their residual (accuracy) and number of iterations (robustness). The benchmark result shows that, under exactly the same conditions, `leastsq` and `lbfgsb` work best, with `leastsq` good at accuracy, while `lbfgsb` good at speed (and theoretically, memory requirement). Thus, I set `leastsq` as default method for the module, but also encourage users to switch to `lbfgsb` in case things turn sour. More details can be seen [here](https://github.com/radis/radis/pull/522#issuecomment-1240720004).

### CODE REVIEW

**Comment by TranHuuNhatHuy** [comment]
In the [latest commit](https://github.com/radis/radis/pull/522/commits/c3dce8e2fedb8fd346620e028aa7680462173045), I added another gallery example which compares performance between [old 1T fitting example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html) and my new fitting module, under exactly the same ground-truths and settings. You can see the improvement based on the comparison result.

I'm not sure this superiority will persist in other cases, but even so, my module is still quite practical and easy to use/apply, right?

**Comment by TranHuuNhatHuy** [comment]
Regarding the latest 2 commits:
- Allow plotting to be completely shut down during a fitting process with `show_plot = False` while calling `fit_spectrum()`. This is ideal for benchmarking process, since showing a plot pauses the runtime, which is not a good thing during benchmarking process when you have to run multiple methods continuously and want to sit back instead of having to close a plot every 3 seconds.
- Add the last gallery example of `plot_newfitting_comparison_methods.py`. This features benchmarking process, result visualization, and which methods are the best choice to be set as default. This benchmarking process targets LMFIT algorithms, compares each one's fitting time and number of fitting loops for a particular spectrum.
For example, here is a benchmarking result:
````
======================== BENCHMARKING RESULT ========================

||           METHOD          ||          RESIDUAL         || LOOPS ||
||---------------------------||---------------------------||-------||
|| leastsq                   || 1.4739494411950239e-07    || 24    ||
|| least_squares             || 1.2170348021620847e-05    || 1     ||
|| differential_evolution    || 1.4739855740762716e-07    || 151   ||
|| brute                     || 1.2287258962300115e-06    || 20    ||
|| basinhopping              || 7.930954059631543e-06     || 151   ||
|| ampgo                     || 4.105104127826488e-07     || 151   ||
|| nelder                    || 1.4739942144030064e-07    || 30    ||
|| lbfgsb                    || 1.4739494411955646e-07    || 28    ||
|| powell                    || 1.473949441200994e-07     || 43    ||
|| cg                        || 1.4776331905574135e-07    || 30    ||
|| cobyla                    || 1.1524288718226295e-05    || 21    ||
|| bfgs                      || 1.4776331905574135e-07    || 30    ||
|| tnc                       || 1.4740393115424221e-07    || 28    ||
|| trust-constr              || 1.4739494411948182e-07    || 26    ||
|| slsqp                     || 1.2170348021620847e-05    || 2     ||
|| shgo                      || 1.0507694502308952e-05    || 5     ||
|| dual_annealing            || 1.5455930218501237e-05    || 151   ||
||---------------------------||---------------------------||-------||
````
And here is the scatter plot:
Screen Shot 2022-09-08 at 22 01 36

Users are encouraged to change the spectrum (and its corresponding ground-truths) for different benchmarking results. My finding is, `leastsq` and `lbfgsb` work best, with `leastsq` good at accuracy, while `lbfgsb` good at speed.

File: examples/plot_newfitting_comparison_oldnew.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+in your JSON, then recall the function again, until the results are satisfied.
+
+This is way easier and more convenient than dwelling into RADIS documentation to find out and
+learn how to use the current example, especially for new RADIS users. This new fitting module
+aims to provide an end-to-end fitting experience, with minimum amount of RADIS knowledge needed.
+
+This example will run both fitting methods on an experimental spectrum with the same ground-truth
+conditions, so you can compare their overall performance, including number of loops, fitting time,
+and final residuals between experimental and best-fit spectra, as an indicator of fit accuracy.
+
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_oldnew.py:25]: Hello

Can we reduce this lengthy text and go straight to the point ? 

For instance, we don't need to know when it was implemented; or to know if it's easier than dwelling into RADIS documnetation (we're already in RADIS documentation!).

Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:25]: I also think the example should **not** compare old & new implmeentaiton of fitting. Users just want to fit. They'll copy/paste this example and adapt it to their case. They're not going to fit twice. 

Can you mention the other implementation in the docstrings, just add a refernece there ("You could also use the old fitting procedure, see ... " ) 

The comparison , however, should be done **for us (developers)**. This Pull Request may be the right place. Just share some performance results in the discussion

File: examples/plot_newfitting_comparison_oldnew.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+# Experimental conditions which will be used for spectrum modeling. Basically, these are known ground-truths.
+experimental_conditions = {
+    "molecule": "CO2",  # Molecule ID
+    "isotope": "1,2",  # Isotope ID, can have multiple at once
+    "wmin": 4167,  # Starting wavelength/wavenumber to be cropped out from the original experimental spectrum.
+    "wmax": 4180,  # Ending wavelength/wavenumber for the cropping range.
+    "wunit": "nm",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
+    "mole_fraction": 1,  # Species mole fraction, from 0 to 1.
+    "pressure": 1 * 1e-3,  # Partial pressure of gas, in "bar" unit.
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_oldnew.py:134]: can this accept astropy units ? If so, do it, it's much safer

File: examples/plot_newfitting_comparison_oldnew.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+
+# ------------------------------------ Step 2. Fill ground-truths and data ------------------------------------ #
+
+
+# Experimental conditions which will be used for spectrum modeling. Basically, these are known ground-truths.
+experimental_conditions = {
+    "molecule": "CO2",  # Molecule ID
+    "isotope": "1,2",  # Isotope ID, can have multiple at once
+    "wmin": 4167,  # Starting wavelength/wavenumber to be cropped out from the original experimental spectrum.
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_oldnew.py:130]: can this accept astropy units ? If so, do it

File: examples/plot_newfitting_comparison_oldnew.py [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+    "wmax": 4180,  # Ending wavelength/wavenumber for the cropping range.
+    "wunit": "nm",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
+    "mole_fraction": 1,  # Species mole fraction, from 0 to 1.
+    "pressure": 1 * 1e-3,  # Partial pressure of gas, in "bar" unit.
+    "path_length": 10,  # Experimental path length, in "cm" unit.
+    "slit": "1.4 nm",  # Experimental slit, must be a blank space separating slit amount and unit.
+    "wstep": 0.001,  # Resolution of wavenumber grid, in cm-1.
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_oldnew.py:137]: using "auto" should be best, no ? What's the default ?

Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:137]: (btw if you want to compare Performances with the other method; it should be the same inputs)

File: examples/plot_newfitting_comparison_oldnew.py [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+bounding_ranges = {
+    "Tgas": [
+        300,
+        2000,
+    ],
+}
+
+# Fitting pipeline setups.
+fit_properties = {
+    "method": "lbfgsb",  # Preferred fitting method from the 17 confirmed methods of LMFIT stated in week 4 blog. By default, "leastsq".
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_oldnew.py:158]: good to mention this ! 

How can we find the other methods ? Can you share this in the docstrings maybe ? 

Btw you can have docstrings appearing nicely as text (with links) in the Gallery example using this format : 

```
# %% 
# Fitting pipeline setups : 
# here this text will render as rst ; including links `like this one <https://...>`__
```

**Comment by erwanp** [comment]
Btw,let's update/fix exisitng GAllery examples from previous PR #520  
- `fit_spectrum()` link doesn't work here https://radis.readthedocs.io/en/latest/auto_examples/plot_newfitting_Tgas.html#sphx-glr-auto-examples-plot-newfitting-tgas-py 
- `fit_spectrum` https://radis.readthedocs.io/en/latest/source/radis.tools.new_fitting.html#radis.tools.new_fitting.fit_spectrum function docstring should include Mini Gallery examples. SEe how it's included in other functions. 


- https://radis.readthedocs.io/en/latest/source/radis.tools.new_fitting.html#radis.tools.new_fitting.spectrum_refinement : i don't understand exactly how to use the function, can you add an Example in the docstrings ? Mayne add link to Gallery example too


- Mention your new fitting routine in See Also part of related radis functions, for instance [SpectrumFactory.fit_spectrum](https://radis.readthedocs.io/en/latest/source/radis.lbl.factory.html#radis.lbl.factory.SpectrumFactory.fit_spectrum)

File: examples/plot_newfitting_comparison_oldnew.py (hunk 1 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:25]: Yes, I will trim that lengthy introduction.

Also, if the user just want to fit, they have the `plot_newfitting_Tgas.py` as well as other 2 examples from Mr. @minouHub and Mr. @CorentinGrimaldi. This comparison example is for comparison between old & new.

Should I remove this old-new example and put up another PR dedicated for it only?

File: examples/plot_newfitting_comparison_oldnew.py (hunk 2 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:134]: For other parameters which are parsed directly into `calc_spectrum()`, yes. You can replace it with astropy units such a s`pressure = 101325 * u.Pa` or `path_length = 0.1 * u.m`.

But how ever, for wavelength/wavenumber, actually not for now, because in my module the `spectrum_refinement()` function crops the original spectrum with:
````python
s_exp.crop(
    wmin,
    wmax,
    wunit,
)
````
Where `wmin`, `wmax` and `wunit` are stated by user.
And for now, as stated in doc, [crop() function](https://radis.readthedocs.io/en/latest/source/radis.spectrum.operations.html#radis.spectrum.operations.crop) only allows 3 units of "nm", "cm-1" and "nm-vac".

So to summarize, astropy units are not applicable for `wmin` and `wmax`, yes for all other.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 4 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:137]: Because originally in the [1-temperature fitting example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py) they use `wstep=0.001`. 

I am comparing the performance between 2 fitting machines, so I must set completely the same experimental inputs.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 5 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:158]: I will share the list of 17 available methods in the original example of `plot_newfitting_Tgas.py` (the simplest one, serving as first example).

Also, thank you for the RST with links. I will apply it throughout my docstrings.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 1 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:25]: In the example here you're fitting another physical case than Minou's one. It's important. Add it.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 2 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:134]: OK ! 
Bonus : can you make crop() read Astropy.units?

File: examples/plot_newfitting_comparison_oldnew.py (hunk 5 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:158]: Add link to lmfit function

File: examples/plot_newfitting_comparison_oldnew.py (hunk 2 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:134]: Now this is a nice topic! I will look into that.
Throughout the development of this new fitting module in the separated repo, I have been restrained by several RADIS limits (please don't take this personal). I hope that I can modify the RADIS codebase and hence my module hereafter.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 1 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:25]: The file you are commenting to, the `plot_newfitting_comparison_oldnew.py`, uses the case of [1-temperature fit example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py).

So now there are only 4 spectra being used:
- A synthetic spectrum, in `plot_newfitting_Tgas.py` and `plot_newfitting_comparison_methods.py`
- Mr. Minou's spectrum, in `plot_newfitting_Tgas-molfrac.py`
- Mr. Corentin's spectrum, in `plot_newfitting_Trot-Tvib-molfrac.py`
- The  [1-temperature fit example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py) spectrum, in `plot_newfitting_comparison_oldnew.py`

Which one do I need to add?

Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_oldnew.py:25]: Oh I misunderstood your point, my apologies Mr. @erwanp . I will replace the spectrum used in this old-new comparison with the one from Mr. @minouHub's case.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 2 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:134]: Of course, that's the idea. You can do it now.

File: examples/plot_newfitting_Tgas.py [hunk 6]
```diff
@@ -61,8 +59,8 @@
     "wmax": 1050,  # Ending wavelength/wavenumber for the cropping range.
     "wunit": "cm-1",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
     "mole_fraction": 0.01,  # Species mole fraction, from 0 to 1.
-    "pressure": 10,  # Partial pressure of gas, in "bar" unit.
-    "path_length": 1,  # Experimental path length, in "cm" unit.
+    "pressure": 10,  # Partial pressure of gas, in "bar" unit by default, but you can also use Astropy units.
```
Comment (erwanp) [inline examples/plot_newfitting_Tgas.py:62]: Warning. This the total pressure not the partial pressure.

Correct this everywhere

File: examples/plot_newfitting_Tgas.py [hunk 7]
```diff
@@ -61,8 +59,8 @@
...
+    "path_length": 1,  # Experimental path length, in "cm" unit by default, but you can also use Astropy units.
```
Comment (erwanp) [inline examples/plot_newfitting_Tgas.py:63]: Show Astropy units example?

File: radis/tools/new_fitting.py [hunk 8]
```diff
@@ -68,6 +68,15 @@ def residual_LTE(params, conditions, s_data, sf, log, verbose):
     residual: float
         residuals of the two spectra, using RADIS's get_residual().
 
+    Examples
+    --------
+    .. minigallery:: radis.tools.new_fitting.residual_LTE
```
Comment (erwanp) [inline radis/tools/new_fitting.py:73]: I think you need a separation line before the double dot statement

**Comment by TranHuuNhatHuy** [comment]
Thank you Mr. @erwanp . Latest commit features:

1. Add `verbose` to every printing command that shows fitting status, such as `currently fetching...`, `successfully doing...` blah blah. By default, `verbose = True`, but if set to `False`, only the fit result will show up. For example:
    - `verbose = True`:
    ````
    ======================= COMMENCE FITTING PROCESS =======================
    
    Successfully retrieved the experimental data in 0.0005199909210205078s.
    
    Acquired spectral quantity 'radiance' from the spectrum.
    NaN values successfully purged.Number of data points left: 16666 points.
    Successfully refined the experimental data in 0.001116037368774414s.
    
    "tol" parameter spotted but "method" is not "lbfgsb"!
    
    Commence fitting process for LTE spectrum!
    
    [[Fit Statistics]]
        # fitting method   = least_squares
        # function evals   = 7
        # data points      = 1
        # variables        = 1
        chi-square         = 2.2403e-14
        reduced chi-square = 2.2403e-14
        Akaike info crit   = -29.4295627
        Bayesian info crit = -31.4295627
    [[Variables]]
        Tgas:  1000.54631 +/- 18.6202380 (1.86%) (init = 700)
    
    Successfully finished the fitting process in 1.3584849834442139s.
    
    ======================== END OF FITTING PROCESS ========================
    ````
    - `verbose = False`: 
    ````
    [[Fit Statistics]]
        # fitting method   = least_squares
        # function evals   = 7
        # data points      = 1
        # variables        = 1
        chi-square         = 2.2403e-14
        reduced chi-square = 2.2403e-14
        Akaike info crit   = -29.4295627
        Bayesian info crit = -31.4295627
    [[Variables]]
        Tgas:  1000.54631 +/- 18.6202380 (1.86%) (init = 700)
    ````

2. Add a blank line after `Examples` in docstring. I just noticed that all of your docstrings are also the same. Some kind of readthedocs rules I guess?

3. Add astropy units to the `plot_newfitting_Tgas.py` example. So now `pressure = 10 bar = 1e6 * u.Pa` and `path_length = 1 cm = 10 * u.mm`, as a demonstration of how astropy units can be used.

4. Correct partial -> total pressure of gas in every reference across all modules and examples.

The `pathLib` or `osPath` will be implemented next.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 4 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:133]: Don't forget to add the performance comparison in the PR test

File: examples/plot_newfitting_Tgas-molfrac.py [hunk 9]
```diff
@@ -48,7 +48,7 @@
     "wmin": 2010.6,  # Starting wavelength/wavenumber to be cropped out from the original experimental spectrum.
     "wmax": 2011.6,  # Ending wavelength/wavenumber for the cropping range.
     "wunit": "cm-1",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
-    "pressure": 1,  # Partial pressure of gas, in "bar" unit by default, but you can use Astropy units too.
+    "pressure": 1,  # Total pressure of gas, in "bar" unit by default, but you can use Astropy units too.
     "path_length": 10,  # Experimental path length, in "cm" unit by default, but you can use Astropy units too.
     "wstep": 0.001,
```
Comment (erwanp) [inline examples/plot_newfitting_Tgas-molfrac.py:53]: wstep shouldn't be given. It's too complicated. Also, it is not a "ground truth" i.e. it's not a physical parameter.

File: examples/plot_newfitting_comparison_oldnew.py (hunk 4 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:133]: And (see my other comment) --> delete wstep ; keep the default; and make sure to use wstep=0.001 but only for the performance comparison. Let's clean the example --> minimal is best

**Review by erwanp — CHANGES_REQUESTED** [review]

**Comment by TranHuuNhatHuy** [comment]
Latest commit includes removal of `wstep` in all examples, except the two comparison/benchmarking examples (`plot_newfitting_comparison_oldnew.py` and `plot_newfitting_comparison_methods.py`). Also, the summary of these benchmarking examples have been put on [this PR's head message](https://github.com/radis/radis/pull/522#issue-1365475821).

I think we are good to merge.

File: examples/plot_newfitting_comparison_algorithm.py [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+function on September 6 2022, we can experience an easier-to-use, robust and complete spectrum
+fitting.
+
+This example features performance comparison between 17 different fitting methods and algorithms
+of LMFIT, introduced in `LMFIT method list <https://lmfit.github.io/lmfit-py/fitting.html#choosing-different-fitting-methods>`.
+There are 2 variables to be compared: the last residual, and total number of loops required for
+the fitting process. A good fitting method should have small number of loops (robustness) and has
+low residual (fitting accuracy).
+
+"""
```
Comment (erwanp) [inline examples/plot_newfitting_comparison_algorithm.py:18]: hello @TranHuuNhatHuy . I'm afraid this takes ages to run (remember it will be run on every new commit ) 

Can you move it somewhere else, like in your repo or in https://github.com/radis/radis-benchmark (manual examples) or in https://github.com/radis/radis-examples ?

File: examples/plot_newfitting_comparison_oldnew.py (hunk 3 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:125]: update

File: examples/plot_newfitting_comparison_oldnew.py (hunk 4 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_oldnew.py:133]: resolve comment if done

File: examples/plot_newfitting_comparison_algorithm.py (hunk 10 above)
Comment (erwanp) [reply examples/plot_newfitting_comparison_algorithm.py:18]: You can mention the example as a link in the docstring of other Gallery examples

File: radis/tools/new_fitting.py [hunk 11]
```diff
@@ -148,10 +158,10 @@ def residual_LTE(params, conditions, s_data, sf, log, verbose):
...
 
     # Print information of fitting process
-    if verbose:
-        for param in params:
-            print(f"{param} = {float(params[param])}")
-        print(f"\nResidual = {residual}\n")
+    # if verbose:
+    #     for param in params:
+    #         print(f"{param} = {float(params[param])}")
+    #    print(f"\nResidual = {residual}\n")
```
Comment (erwanp) [inline radis/tools/new_fitting.py:164]: if it's commented, just remove the lines

File: radis/tools/new_fitting.py [hunk 12]
```diff
@@ -278,10 +298,10 @@ def residual_NonLTE(params, conditions, s_data, sf, log, verbose):
...
 
     # Print information of fitting process
-    if verbose:
-        for param in params:
-            print(f"{param} = {float(params[param])}")
-        print(f"\nResidual = {residual}\n")
+    # if verbose:
+    #     for param in params:
+    #         print(f"{param} = {float(params[param])}")
+    #     print(f"\nResidual = {residual}\n")
```
Comment (erwanp) [inline radis/tools/new_fitting.py:304]: same

**Review by erwanp — CHANGES_REQUESTED** [review]

File: examples/plot_newfitting_comparison_algorithm.py (hunk 10 above)
Comment (TranHuuNhatHuy) [reply examples/plot_newfitting_comparison_algorithm.py]: I have just deleted the file here, and add it to `radis-benchmark` through a [PR](https://github.com/radis/radis-benchmark/pull/13).

When you approve the PR, I will add its link to all other example files.

**Comment by TranHuuNhatHuy** [comment]
Mr. @erwanp ,

In the latest commits, I have:
- Adapted new fitting modules with astropy units.
- Added astropy units into example files.
- Cleaned all comment blocks.

Now, upon your approval of my [benchmark PR](https://github.com/radis/radis-benchmark/pull/13), I will add its link to all the example files. For the rest, I think we are good to go.

**Review by erwanp — APPROVED** [review]
Lgtm, you can merge!
